api/services/prompt_service.py

```diff
@@ -171,7 +171,7 @@ def update_mapped_fields(
     def sanitize_keypress(self, data):
         keypress = data["keypress"]
         try:
-            if keypress:
+            if keypress and len(keypress) < 5:
```

Comment:
These changes were for avoiding webhook failure due to keypress value with more than 4 digits.